webpack.config.js

```diff
@@ -0,0 +1,47 @@
+const path = require('path')
+const webpack = require('webpack')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const CleanWebpackPlugin = require('clean-webpack-plugin')
+
+module.exports = {
+  context: path.resolve(__dirname, 'src'),
+  entry: {
+    app: './app.js'
+  },
+  output: {
+    filename: '[name].bundle.js',
+    path: path.resolve(__dirname, 'dist')
+  },
```

Comment:
Don't we want `devtool: 'source-map'` as well?

Reply:
Yes :)